src/test/java/dbProcs/GetterTest.java

```diff
@@ -3144,6 +3144,7 @@ public void testFindAdminById()
 	
 	@Test
 	public void testGetAdmins()
+	//TODO

```

Comment:
Is there somthing you were planning to add here?

Reply:
Missed it. I've removed it